analytics_data_api/v0/tests/views/test_courses.py

```diff
@@ -6,15 +6,15 @@
 import datetime
 from collections import OrderedDict
 from itertools import groupby
+from unittest.mock import Mock, patch
 
 import ddt
 import pytz
 import six
-import six.moves.urllib.parse  # pylint: disable=import-error
+import six.moves.urllib.parse  # pylint: disable=import-error,wrong-import-order
```

Comment:
You may be able to drop this line completely, it seems that `import six` above should already cover this.